workers/data_refinery_workers/processors/smasher.yml

```diff
@@ -0,0 +1,18 @@
+---  # "Illumina SCAN" processor
+
+# Operating system distribution (such as Ubuntu 16.04, CentOS 7.2, etc)
+os_distribution:   # DO NOT EDIT this line
+
+# OS-level packages (installed by OS package manager)
+os_pkg:
+  - python3
+  - python3-pip
+
+# Python pip packages
+python:
+  - Django
+  - data-refinery-common
+  - matplotlib
```

Comment:
I added it because `matplotlib` was listed in `processors/requirements.in` and imported by `test_smasher.py`. But I just realized that it was not being used at all. I will remove it from all files that include this package.